common/get-solidity-protos.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+WEAVER_ROOT=${1:-".."}
+
+BESU_PROTOSDIR=${WEAVER_ROOT}/common/besu-protos
```

Comment:
Whatever you are downloading or creating in this folder is not specific to Besu, right? So I'd suggest renaming this folder and moving it as well as this script file into `common/protos-solidity`. This logic doesn't need to be in the root `common` folder.